checker/manual/external-tools.tex

```diff
@@ -302,17 +302,17 @@ \subsection{Explanation\label{ant-task-explanation}}
 \section{Maven\label{maven}}
```

Comment:
Please have a look at the demo project at https://github.com/renatoathaydes/checker-maven-demo

It definitely works.